frontend/packages/console-app/src/components/console-operator/ConsoleOperatorConfig.tsx

```diff
@@ -212,3 +252,8 @@ type ConsolePluginStatusType = {
 type ConsolePluginsListType = {
   obj: K8sResourceKind;
 };
+
+type ConsolePluginsListResponseType = {
+  consoleCommit?: string;
+  plugins?: any;
```

Comment:
This is an array of strings, right?

```suggestion
  plugins?: string[];
```